### DISCUSSION TITLE
A way to get the player's saved hotkeys.

### DISCUSSION THREAD

**Discussion by DanielWillett** [SmartlyDressedGames/Unturned-3.x-Community#3737] [Ideas]
`ServerBindItemHotkey` and `ServerClearItemHotkey` are nice, but it's hard to find a good use for them without being able to know what the player already has binded and when it changes... for example my use case would be to save and load different hotkeys for loadouts/kits, but to use it, players would have to set their keybinds with a command.

**Comment by Corbyncc** [comment]
+1 Would like to see this feature. Makes sense to add it and not sure why it was overlooked when these methods were added.

**Comment by Corbyncc** [comment]
Willing to do questionable things both morally and otherwise to get this added :)

**Comment by Jdance-Media** [comment]
👉👈

**Comment by liebesleid-eth** [comment]
Pretty please ;-;

**Comment by ghost** [comment]
Any update on this? It's quite easy to implement.
